tmail-backend/apps/distributed/src/test/resources/blob.properties

```diff
@@ -0,0 +1,87 @@
+
+# ============================================== ObjectStorage ============================================
+
+# ========================================= ObjectStorage Buckets ==========================================
+# bucket names prefix
+# Optional, default no prefix
+# objectstorage.bucketPrefix=prod-
+
+# Default bucket name
+# Optional, default is bucketPrefix + `default`
+# objectstorage.namespace=james
+
+# ========================================= ObjectStorage on S3 =============================================
+# Mandatory if you choose s3 storage service, S3 authentication endpoint
+objectstorage.s3.endPoint=http://s3.docker.test:8000/
+
+# Mandatory if you choose s3 storage service, S3 region
+#objectstorage.s3.region=eu-west-1
+objectstorage.s3.region=us-east-1
+
+# Mandatory if you choose aws-s3 storage service, access key id configured in S3
+objectstorage.s3.accessKeyId=accessKey1
+
+# Mandatory if you choose s3 storage service, secret key configured in S3
+objectstorage.s3.secretKey=secretKey1
+
+# ============================================ Blobs Exporting ==============================================
+# Read https://james.apache.org/server/config-blob-export.html for further details
+
+# Choosing blob exporting mechanism, allowed mechanism are: localFile, linshare
+# LinShare is a file sharing service, will be explained in the below section
+# Optional, default is localFile
+blob.export.implementation=localFile
+
+# ======================================= Local File Blobs Exporting ========================================
+# Optional, directory to store exported blob, directory path follows James file system format
+# default is file://var/blobExporting
+blob.export.localFile.directory=file://var/blobExporting
+
+# ======================================= LinShare File Blobs Exporting ========================================
+# LinShare is a sharing service where you can use james, connects to an existing LinShare server and shares files to
+# other mail addresses as long as those addresses available in LinShare. For example you can deploy James and LinShare
+# sharing the same LDAP repository
+# Mandatory if you choose LinShare, url to connect to LinShare service
+# blob.export.linshare.url=http://linshare:8080
+
+# ======================================= LinShare Configuration BasicAuthentication ===================================
+# Authentication is mandatory if you choose LinShare, TechnicalAccount is need to connect to LinShare specific service.
+# For Example: It will be formalized to 'Authorization: Basic {Credential of UUID/password}'
+
+# blob.export.linshare.technical.account.uuid=Technical_Account_UUID
+# blob.export.linshare.technical.account.password=password
+
+# When deduplication is active, use a Cassandra projection to avoid saving several time the same blob to S3.
+# This prevents isolation issues for Swift backed S3 deployments and also  adds a performance boost.
+single.save.enable=true
```

Comment:
Please move this in `# ========================================= ObjectStorage deduplication ========================================` section

Reply:
Can you add the option in sample-configuration too?